### DISCUSSION TITLE
Enable stack's gap responsiveness

### DISCUSSION THREAD

**Discussion by AlioshaRabeshko** [mantinedev/mantine#5645] [Q&A]
Hello. I've been using stack components in order to organize consistent gaps between inputs. The problem is that for small screens the form changes its layout and gaps that used to be perfect for large screens, don't suit the small ones.
I think it would be a useful feature to support responsiveness like many other props(eg margins, padings, gutters etc), lets say {base: 'xs', sm: 'lg'}

Or is there any other way to change the gap responsively?

**Comment by AlioshaRabeshko** [comment]
@rtivital can we add the gap parameter into extractStyleProps? It looks like a style prop after all. Or is there a reason why it is not added there?

**Reply by rtivital** [reply]
New features are added in minor releases. I've added this to the backlog. It may be added in one of the next minor releases.
